galata/test/jupyterlab/notebook-mobile.test.ts

```diff
@@ -14,6 +14,8 @@ test.describe('Notebook Layout on Mobile', () => {
 
   test('Execute code cell', async ({ page }) => {
     await page.sidebar.close('left');
+    // TODO: calling `setCell` just once leads to very flaky test
```

Comment:
Maybe we can link to the issue?

```suggestion
    // TODO: calling `setCell` just once leads to very flaky test
    // See https://github.com/jupyterlab/jupyterlab/issues/15252 for more information
```